app/controllers/patients_controller.rb

```diff
@@ -236,8 +236,10 @@ def update
 
     # Update patient history with detailed edit diff
     patient_before = patient.dup
-    Patient.detailed_history_edit(patient_before, patient, allowed_params&.keys, current_user.email) if patient.update(content)
 
+    render(json: patient.errors, status: 422) && return unless patient.update(content)
+
+    Patient.detailed_history_edit(patient_before, patient, allowed_params&.keys, current_user.email) if patient.update(content)
```

Comment:
Wondering if we should revert this now. I guess before it was just relying on front end validation catching everything the model validations would. If it didn't, I guess it would have just silently failed in the UI? I think my preference would be to leave it as it was, since the 422 isn't a great UX either, and then come back and improve it when we implement the follow up task for unifying the UI with all the backend validations.

Reply:
I see what you mean, but I do think this is a desired change I'd like to include sooner rather than later. As it was before this change, it would fail silently and go on to update history which was just incorrect. The frontend does catch this and show the error code in a toast, so at least we're returning an accurate code back and not incorrectly acting as if the update was successful.

Reply:
Gotcha, sounds good to me then